src/com/sap/piper/integration/TransportManagementService.groovy

```diff
@@ -44,6 +44,12 @@ class TransportManagementService implements Serializable {
             ]
         ]
 
+        def proxy = config.proxy ? config.proxy : script.env.HTTP_PROXY
```

Comment:
Is it sufficient to check `script.env.HTTP_PROXY`?  `env` is to my knowledge an instance of `EnvActionImpl` and `env.XXX` looks like a shortcut to `env.getProperty('XXX')`. Can we be sure that this property has been set on `env`? What happens if that is "only" set on the level of the environment variables of the asscociated shell? I guess these environment variables are exposed via `env.getEnvironment()`.

I don't say that the current approach does not work / is wrong / is not sufficient. But I'm not 100% sure. Maybe someone else can cross check.